tutorials/application_design/c++11/common.hpp

```diff
@@ -18,71 +18,75 @@
 
 #include "VehicleModeling.hpp"
 
-namespace utils {
-
-double random_range(double min, double max)
-{
-    struct SrandInit {
-        explicit SrandInit(int seed) : seed_(seed)
-        {
-            std::srand(seed_);
-        }
-
-        int seed_;
-    } static const srand_init(std::time(nullptr));
+using CoordSequence = rti::core::bounded_sequence<Coord, 100>;
 
-    return min + (max - min) * ((double) std::rand() / (RAND_MAX));
-}
+namespace {  // Coord namespace
 
-double random_stduniform()
+std::string to_string(const Coord &coord)
 {
-    return random_range(0.0, 1.0);
+    std::ostringstream ss;
+    ss << "Coord(lat: " << coord.lat() << ", lon: " << coord.lon() << ")";
+    return ss.str();
 }
 
-std::string new_vin()
+}  // namespace
+
+namespace rti::core {  // bounded_sequence namespace
+
+std::string to_string(const CoordSequence &route)
 {
-    static const std::string choices("0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ");
+    using ::to_string;
 
     std::ostringstream ss;
-    for (int i = 0; i < VIN_LENGTH; ++i) {
-        ss << choices[random_range(0, choices.size())];
+    ss << "Route(";
+    for (auto it = route.begin(); it != route.end();) {
+        ss << to_string(*it);
+        if (++it != route.end()) {
+            ss << ", ";
+        }
     }
+    ss << ")";
     return ss.str();
 }
 
-using CoordSequence = rti::core::bounded_sequence<Coord, 100>;
+}  // namespace rti::core
+
+namespace utils {
+
+namespace details {
+
+static std::random_device rd;
+static std::mt19937 gen { rd() };
+
+};  // namespace details
 
-template<typename T>
-std::string to_string(const T &v)
+void set_random_seed(unsigned seed)
```

Comment:
I believe the seed was already randomly set by 
```
    std::random_device rd;
    std::mt19937 gen(rd());
```

So this function is likely not necessary

Reply:
▶️  TODO in new PR